torchtitan/config_manager.py

```diff
@@ -450,6 +450,13 @@ class Float8:
     Example: --float8.filter_fqns "attention.wq,attention.wk,attention.wv,output"
     """
 
+    emulate: bool = False
+    """
+    Whether to run on earlier hardware in CI test.
+    torch.compile with float8 dtypes is not going to work on older hardware, so the emulation can
+    only be used in eager mode.
```

Comment:
```suggestion
    If True, emulation is used instead of hardware accelerated gemm. This is for test purpose only, as the current CI does have sm_90 capability, required by Float8.
    Not compatible with torch.compile.
```
This is assuming torch.compile+emulate don't work on >= H100 either. If not we'll need to further adjust code and helper message.